server/templates/pages/src/index.html

```diff
@@ -36,17 +36,18 @@
         <div id="fox-logo"></div>
         <div id="stage">
           <div id="main-content" class="card">
-              <!--[if lt IE 10]>
-                  <p class="error browsehappy">
-                    {{#t}}You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.{{/t}}</p>
-              <![endif]-->
-              <!-- this will be overwritten if JS is enabled -->
-              <noscript>
-                {{#t}}Firefox Accounts requires JavaScript.{{/t}}
-              </noscript>
+            <!--[if lt IE 10]>
```

Comment:
The browse happy area no longer shows in IE9:

![screen shot 2015-03-29 at 13 05 51](https://cloud.githubusercontent.com/assets/848085/6885357/bd0425a2-d614-11e4-98dd-749019c252af.png)

But it does in prod:
![screen shot 2015-03-29 at 13 07 54](https://cloud.githubusercontent.com/assets/848085/6885359/c3563b70-d614-11e4-904a-8e9ac5c995f9.png)